h2o-core/src/main/java/water/jdbc/SQLManager.java

```diff
@@ -269,26 +306,26 @@ public ConnectionPoolProvider() {} // Externalizable classes need no-args constr
 
     }
 
-    /**
-     * @return Number of connections to an SQL database to be opened on a single node.
-     */
-    static int getMaxConnectionsPerNode(final int cloudSize, final short nThreads, final int nChunks) {
-      int conPerNode;
+    static int getMaxConnectionsTotal() {
+      int maxConnections = MAX_CONNECTIONS;
       final String userDefinedMaxConnections = System.getProperty(MAX_USR_CONNECTIONS_KEY);
       try {
-        Integer connectionAmount = Integer.valueOf(userDefinedMaxConnections);
-        if (connectionAmount > 0 && connectionAmount < MAX_CONNECTIONS) {
-          conPerNode = calculateLocalConnectionCount(connectionAmount, cloudSize, nThreads, nChunks);
-        } else {
-          conPerNode = calculateLocalConnectionCount(MAX_CONNECTIONS, cloudSize, nThreads, nChunks);
+        Integer userMaxConnections = Integer.valueOf(userDefinedMaxConnections);
+        if (userMaxConnections > 0 && userMaxConnections < MAX_CONNECTIONS) {
+          maxConnections = userMaxConnections;
         }
       } catch (NumberFormatException e) {
         Log.info("Unable to parse maximal number of connections: " + userDefinedMaxConnections
-            + ". Falling back to default settings.");
-        conPerNode = calculateLocalConnectionCount(MAX_CONNECTIONS, cloudSize, nThreads, nChunks);
+                + ". Falling back to default settings.");
```

Comment:
This part was written be me and I'd like to enhance it a little bit. Would it be possible to add the number of connections we're backing up to ? Just concat the `MAX_CONNECTIONS` constraint ?

Reply:
sure